docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -884,10 +891,30 @@ volume on the first run.
 
 The path of a device which is mounted for the volume.
 
+### `Driver=`
+
+Specify the volume driver name. When set to `image`, the `Image` key must also be set.
+
+This is equivalent to the Podman `--driver` option.
+
 ### `Group=`
 
 The host (numeric) GID, or group name to use as the group for the volume
 
+### `Image=`
+
+Specifies the image the volume is based on when `Driver` is set to the `image`.
+It is recommended to use a fully qualified image name rather than a short name, both for
```

Comment:
Non-blocking nit: we could point to the `podman-pull` man page for details (as the log message does)